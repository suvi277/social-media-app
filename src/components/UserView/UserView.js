import React from 'react';
import { Link } from 'react-router-dom';
import './UserView.scss';

const UserView = ({ email, firstName, lastName, id }) => {
	return (
		<div className="user-list-row">
			<Link to={`/profile/${id}`}>
				<h5>{`${firstName} ${lastName}`}</h5>
			</Link>
			<span className="small">{email}</span>
		</div>
	);
};

export { UserView };
