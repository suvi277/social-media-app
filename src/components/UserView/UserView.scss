.user-list-row {
    margin-bottom: 12px;
    border-bottom: 1px solid #cccccc;
    padding: 20px 0;
}