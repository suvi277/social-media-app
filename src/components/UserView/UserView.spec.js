import React from 'react';
import { shallow } from 'enzyme';
import { UserView } from './UserView';

describe('UserView Component', () => {
	const wrapper = shallow(<UserView />);

	it('renders as expected', () => {
		expect(wrapper).toMatchSnapshot();
	});
	//To Complet Test Cases
});
