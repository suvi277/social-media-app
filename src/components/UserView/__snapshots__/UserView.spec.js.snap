// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserView Component renders as expected 1`] = `ShallowWrapper {}`;
